index.js

```diff
@@ -110,8 +119,10 @@ module.exports = function(options) {
         finalObject[decomposition[1]] = {};
       }
 
+      logDebug('scopeTest returned true for: ' + name);
       defineProperty(finalObject[decomposition[1]], getRequireName(decomposition[2]), name);
     } else {
+      logDebug('scopeTest returned false for: ' + name);
```

Comment:
From the looks of things, all the changes after line 100 in this file are safe to be reverted. No logic has actually changed.